### ISSUE TITLE
Release 2.6.2

### ISSUE THREAD

**Issue by camilasan** [nextcloud/desktop#1634]
## 🔮 The plan
- [x] RC1: TBD
- [x] Final Release: after two weeks time (max) of latest RC release.

### 📝 TO DO
- [ ] [Milestone: 2.6.2](https://github.com/nextcloud/desktop/milestone/13)
- [ ] 👥 Pull requests:
     - [ ] [label:backport-request](https://github.com/nextcloud/desktop/issues?q=is%3Aopen+is%3Apr+label%3Abackport-request) 
- [ ] When needed: backport master merged PR's using the backport bot by commenting:
```
/backport to stable-2.6
```
- [x] Create tag from branch ```stable-2.6``` created previously
```
$ git checkout stable-2.6
$ git tag -s v2.6.2-rc1 -m "Release..."
$ git push origin --tags
```
- [x] 🏭 Login on the build servers to change the scripts with the current tag and version suffix to build the release:
    - [x] via Parallels for the Mac OS
    - [x] via RDP for Windows
    - [x] via ssh for Linux      
- [x] Move new pkg/exe/AppImage it to [download.nextcloud.com/desktop/releases](https://download.nextcloud.com/desktop/releases/)
- [x] Get changelog using [github_helper scripts](https://github.com/nextcloud/github_helper/tree/master/changelog)
- [x] Create pre-release/release at [nextcloud/desktop/releases](https://github.com/nextcloud/desktop/releases) with the new changelog.
- [ ] 🔔 Notify Marketing to share the release in the proper channels.

### ⚠️ TO DO before major releases
- [ ] 🍴 Create stable branch (e.g. ```stable-2.6```) after tagging RC1 for major release (e.g. ```v2.6.2-rc1```)
- [ ] Update ```CHANGELOG.md``` with the new changelog.
- [ ] Bump version in ```master``` in [VERSION.cmake](https://github.com/nextcloud/desktop/blob/master/VERSION.cmake) (e.g. ```2.7.0```)

**Comment by ivaradi** [comment]
@camilasan Could we somehow make tagging to trigger a build so that the release could be built automatically? According to my experiments, it is a simple setting of the Drone server. Could someone have a look at it?

(I don't mind performing a manual build of the release, since I already have a script to do that, but IMHO it would be nicer if the Drone server could do it automatically.)

**Comment by er-vin** [comment]
I guess this can be closed now, 2.6.2 has been released
